var/spack/repos/builtin/packages/qt-base/package.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class QtBase(CMakePackage):
+    """Qt Base (Core, Gui, Widgets, Network, ...)"""
+
+    homepage = "https://www.qt.io"
+    url      = "https://github.com/qt/qtbase/archive/refs/tags/v6.2.3.tar.gz"
+    list_url = "https://github.com/qt/qtbase/tags"
+
+    maintainers = ['wdconinc', 'sethrj']
+
+    version('6.2.4', sha256='657d1405b5e15afcf322cc75b881f62d6a56f16383707742a99eb87f53cb63de')
+    version('6.2.3', sha256='2dd095fa82bff9e0feb7a9004c1b2fb910f79ecc6111aa64637c95a02b7a8abb')
+
+    generator = 'Ninja'
+
+    # Changing default to Release for typical use in HPC contexts
+    variant('build_type',
+            default='Release',
+            values=("Release", "Debug", "RelWithDebInfo", "MinSizeRel"),
+            description='CMake build type')
+
+    variant('accessibility', default=True, description='Build with accessibility support.')
+    variant('dbus', default=False, description='Build with D-Bus support.')
+    variant('examples', default=False, description='Build examples.')
+    variant('framework', default=False, description='Build as a macOS Framework package.')
+    variant('gtk', default=False, description='Build with gtkplus.')
```

Comment:
This variant might only be applicable when `+widgets`? It also is inapplicable to macos.